ambry-protocol/src/main/java/com/github/ambry/protocol/LogInfo.java

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright 2019 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.protocol;
+
+import com.github.ambry.utils.Utils;
+import io.netty.buffer.ByteBuf;
+import java.io.DataInputStream;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.util.ArrayList;
+import java.util.List;
+
+public class LogInfo {
+  private String fileName;
+  private long fileSizeInBytes;
+  List<FileInfo> listOfIndexFiles;
+  List<FileInfo> listOfBloomFilters;
```

Comment:
They are already lists, we don't need to put list in the variable names anymore. Could you just rename it to indexFiles and bloomFilterFiles?

Reply:
done